lib/utils/preferences.dart

```diff
@@ -65,6 +66,8 @@ class Preferences {
     _log.info("set payment options channel setup limit fee : $channelFeeLimitMsat");
     final prefs = await SharedPreferences.getInstance();
     await prefs.setInt(_kPaymentOptionChannelSetupFeeLimit, channelFeeLimitMsat);
+    // iOS Extension requirement
+    await SharedPreferenceAppGroup.setInt(_kPaymentOptionChannelSetupFeeLimit, channelFeeLimitMsat);
```

Comment:
Is there a need to set it twice? We could also get the preference in the same way using `SharedPreferenceAppGroup.get(_kPaymentOptionChannelSetupFeeLimit)`

Reply:
Makes sense now sorry, it's iOS only

Reply:
Yes, this is workaround for not being able to access `shared_preferences` values on iOS extension. Values set through `shared_preference_app_group` live on app group's UserDefaults and do not overlap with `shared_preferences` values.